scripts/zones/Eastern_Adoulin/npcs/Octavien.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------
+-- Area: Eastern Adoulin (257)
+--  NPC: Octavien
+-- Type: Palace Guard
+-- Starts Children of the Rune
+-- !pos 100.580 -40.150 -63.830
+-----------------------------------
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_utils.lua")
+require("scripts/globals/quests")
+local ID = require("scripts/zones/Eastern_Adoulin/IDs")
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    -- CHILDREN OF THE RUNE
+    local childrenOfTheRuneQuestStatus = player:getQuestStatus(ADOULIN, tpz.quest.id.adoulin.CHILDREN_OF_THE_RUNE)
+    if and childrenOfTheRuneQuestStatus == QUEST_AVAILABLE and player:getMainLvl() >= ADVANCED_JOB_LEVEL then
```

Comment:
`if and` oopsie?

Reply:
Yep. Was a typo. I had a few others too. Couldn't test out the latest changes until my client updated.